apps/daimo-mobile/src/view/shared/AmountInput.tsx

```diff
@@ -78,6 +78,7 @@ export function AmountInput({
       keyboardType="numeric"
       placeholder="0"
       placeholderTextColor={color.gray}
+      multiline
```

Comment:
hmm so the AmountInput was multiline before, and we removed it due to an issue on iOS 

i'll test it again and add more detail

it's a weird fix in any case, because AmountInput is single line